src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -45,14 +45,14 @@ public AddCommand parse(String args) throws ParseException {
         }
 
         argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_NAME, PREFIX_PHONE, PREFIX_EMAIL, PREFIX_ADDRESS, PREFIX_JOB,
-                PREFIX_INCOME);
+                PREFIX_INCOME, PREFIX_TIER);
         Name name = ParserUtil.parseName(argMultimap.getValue(PREFIX_NAME).get());
         Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
         Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
         Address address = ParserUtil.parseAddress(argMultimap.getValue(PREFIX_ADDRESS).get());
         Job job = ParserUtil.parseJob(argMultimap.getValue(PREFIX_JOB).get());
         Income income = ParserUtil.parseIncome(argMultimap.getValue(PREFIX_INCOME).get());
-        Tier tier = ParserUtil.parseTag(argMultimap.getValue(PREFIX_TAG).orElse("NA"));
+        Tier tier = ParserUtil.parseTag(argMultimap.getValue(PREFIX_TIER).orElse("NA"));
```

Comment:
Small thing to take note of, I have since saved this constant in a class of its own. No action needed from you.

Reply:
👍